pinot-query-runtime/src/main/java/org/apache/pinot/query/service/QueryConfig.java

```diff
@@ -23,13 +23,13 @@
  */
 public class QueryConfig {
   public static final String KEY_OF_QUERY_SERVER_PORT = "pinot.query.server.port";
-  public static final int DEFAULT_QUERY_SERVER_PORT = -1;
+  public static final int DEFAULT_QUERY_SERVER_PORT = 0;
 
   public static final String KEY_OF_QUERY_RUNNER_HOSTNAME = "pinot.query.runner.hostname";
   public static final String DEFAULT_QUERY_RUNNER_HOSTNAME = "localhost";
   // query runner port is the mailbox port.
   public static final String KEY_OF_QUERY_RUNNER_PORT = "pinot.query.runner.port";
-  public static final int DEFAULT_QUERY_RUNNER_PORT = -1;
+  public static final int DEFAULT_QUERY_RUNNER_PORT = 0;
```

Comment:
Can we specify two default runner ports, one for broker and the other one for server? By doing so, you can just reuse the default port numbers (i.e. `8421` and `8442`) for both broker and server.

Reply:
i am not sure this is the right solution b/c runner ports are not specific to the server or broker. 
intermediate query runners can run on either broker or server, we just happened to only utilize the server component.

Reply:
for now I think we can just leave it since this is a config file. the comment is already there in QueryRunner